jwst/datamodels/schemas/nrsifu_apcorr.schema.yaml

```diff
@@ -0,0 +1,201 @@
+%YAML 1.1
+---
+$schema: "http://stsci.edu/schemas/asdf/asdf-schema-1.0.0"
+id: "http://stsci.edu/schemas/jwst_datamodel/nrsifu_apcorr.schema"
+title: NRS Ifu aperture correction data model
+allOf:
+- $ref: referencefile.schema
+- $ref: keyword_exptype.schema
+- $ref: keyword_pexptype.schema
+- type: object
+  properties:
+    apcorr_table:
+      type: object
+      properties:
+          grating:
+            type: string
+          filter:
+            type: string
+          wavelength:
+            datatype: float32
+          wavelength_units:
+            type: string
+          radius:
+            datatype: float32
+          radius_units: 
+            type: string
+          apcorr:
+            datatype: float32
+          apcorr_err:
+            datatype: float32
+    apcorr_table_clear_prism:
+      type: object
+      properties:
+          grating:
+            type: string
+          filter:
+            type: string
+          wavelength:
+            datatype: float32
+          wavelength_units:
+            type: string
+          radius:
+            datatype: float32
+          radius_units: 
+            type: string
+          apcorr:
+            datatype: float32
+          apcorr_err:
+            datatype: float32
+    apcorr_table_f070lp_g140h:
```

Comment:
So at one point we thought about pulling out filter/grating and having a different reference file for each. We ended up just leaving it in the code mainly because I was not sure how CRDS would handle the cases of multiple bands, channels, gratings or filters. After testing I think 'MULTIPLE' is set if that is the case - so we can go ahead and break then up into separate reference files. NOTHING has been loaded into CRDS.